src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/DockerHelper.groovy

```diff
@@ -38,6 +38,24 @@ class DockerHelper {
         dockerClient.waitContainer(containerId)
     }
 
+    String getId(ContainerName containerName) {
```

Comment:
minor suggestion: good to align method names to the same format, so if we using getContainerIp to use getContainerId, or
to use simple getId and getIp (in this case we can use getAddress to avoid misstyping)

Reply:
Fair enough.